src/main/scala/net/cakesolutions/CakeBuildPlugin.scala

```diff
@@ -58,13 +65,23 @@ object CakeBuildPlugin extends AutoPlugin {
       sys.env.get("SBT_VOLATILE_TARGET") match {
         case None => target.value
         case Some(base) =>
+          val baseDir = new File(base)
+          assert(
```

Comment:
also, by the time this code is executing, the `AutoPlugin` has already loaded so I don't see what the point of wrapping the assertion error in another exception is.